reference/carts.v3.yml

```diff
@@ -1749,15 +1748,12 @@ components:
                     description: 'Combined download size, in human-readable style. E.g.: `30MB`.'
```

Comment:
Recommend the following for description: 'Specifies the combined download size of all files, in human-readable style, for example: `30MB`.'